src/tests/functional/plugin/shared/include/behavior/plugin/ov_auto_batching_tests.hpp

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include <utility>
+#include <vector>
+#include <memory>
+
+#include "base/ov_behavior_test_utils.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "common_test_utils/subgraph_builders/single_conv.hpp"
+#include "common_test_utils/subgraph_builders/detection_output.hpp"
+#include "common_test_utils/subgraph_builders/multi_single_conv.hpp"
+#include "functional_test_utils/plugin_cache.hpp"
+
+namespace ov {
+namespace test {
+namespace behavior {
+using AutoBatchTwoNetsParams = std::tuple<
+        std::string,  // device name
+        bool,         // get or set tensor
+        size_t,       // number of streams
+        size_t,       // number of requests
+        size_t>;      // batch size>
+
+class AutoBatching_Test : public OVPluginTestBase,
+                          public testing::WithParamInterface<AutoBatchTwoNetsParams> {
+public:
+    void SetUp() override {
```

Comment:
Can be moved to protected part, if used only in test cases and `testing::Test` define it as protected also.

Reply:
fixed